subprojects/docs/src/docs/userguide/cpp_testing.adoc

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[cpp_testing]]
+= Testing in C++ projects
+
+////
+To run integration tests:
+
+   ./gradlew :integTest:intTestImage :docs:extractSamples :integTest:integTest --tests "*SamplesCppTestingIntegrationTest"
+////
```

Comment:
Do you have some samples that should have been part of this?

Reply:
This is actually out of date. It was copied from Testing Java which I think is also out-of-date.